### PR TITLE
chore: delete old flake8 config file

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] ~Added a ``news`` entry~ (skipped for this PR)
* [x] Regenerated schema JSON if schema altered (`python conda_smithy/schema.py`)

We do not need this one anymore.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge/core this is ready for review! I skipped the news entry.

File: .flake8 [hunk 1]
```diff
@@ -1,3 +0,0 @@
-[flake8]
-max-line-length = 100
```
Comment (hmaarrfk) [inline .flake8:2]: `pyproject.toml` specifies 79 line length

File: .flake8 [hunk 2]
```diff
@@ -1,3 +0,0 @@
...
-extend-ignore = E203
```
Comment (hmaarrfk) [inline .flake8:3]: is this OK to not have a line ignoring this in pyproject.toml

**Review by hmaarrfk — APPROVED** [review]
two small questions. merge at will likely since they aren't triggering any issues on CIs.

File: .flake8 (hunk 2 above)
Comment (beckermr) [reply .flake8:3]: Yes. We moved to ruff + black (or ruff + ruff format, I don't recall), so the line length is not relevant anymore.

**Pull request merged**